arcade/scene.py

```diff
@@ -307,7 +313,7 @@ def draw(self, names: Optional[List[str]] = None, **kwargs) -> None:
         If `names` parameter is provided then only the specified SpriteLists
         will be drawn. They will be drawn in the order that the names in the
         list were arranged. If `names` is not provided, then every SpriteList
-        in the scene will be drawn according the order of the main sprite_lists
+        in the scene will be drawn according the order of the main _sprite_lists
```

Comment:
Should probably be phrased wiout mentioning the internal member.

Reply:
change "the order of the main sprite_lists" to "the order of the sprite lists"